crates/matrix-sdk/tests/integration/encryption.rs

```diff
@@ -0,0 +1,237 @@
+use std::{
```

Comment:
Can you put ~all of this code into a `verification` submodule? The secret storage PR already adds a `secrets_storage.rs` submodule: https://github.com/matrix-org/matrix-rust-sdk/pull/2621/files#diff-cc6e326da953cebbe55173940c6bda458f880aaabbf06e5bbd869f62f1afb1b8R1-R3

Should make the merge conflict easier to handle.

Reply:
Sure!